portal/src/components/Header/components/MainMenu.scss

```diff
@@ -113,10 +113,10 @@
         &:focus {
             color: jkl.$color-granitt;
 
-            transform: translateX(-1ch);
+            transform: translateX(jkl.rem(-2px));
```

Comment:
Nå vil vel ikke bevegelsen skalere med fontstørrelsen, men det er kanskje greit?

Reply:
Det er meningen 😅 Når denne var ganske stor (for eksempel ved 200% zoom) ble bevelgelsen så stor at den gikk inn og ut av hover-tilstanden. Så ganske rart ut, og ble praktisk talt umulig å klikke på.

Reply:
Haha, ok 😅